CHANGES.rst

```diff
@@ -7,6 +7,7 @@ Version 0.9.0 (UNRELEASED)
 - Adds REANA specification validation and loading logic from ``reana-client``.
 - Adds support for networking/v1 API to Kubernetes Python client.
 - Adds new ``/api/launch`` endpoint that allows running workflows from remote sources.
+- Changes REANA specification loading functionality to support remote working directories.
```

Comment:
I am not sure "support remote working directories" is accurate. Cause, we do not really work with remote directories.

*suggestion:* Maybe, change to:

```
- Changes REANA specification loading functionality to support changing working directories.
```

Reply:
Thanks, good point, it was indeed confusing, what about:
```
- Changes REANA specification loading functionality to allow specifying different working directories.
```
WDYT?